### PR TITLE
Doc: Add filter known issue to 7.15.2 release notes

### PR DESCRIPTION
**Pull request by karenzone**
Add content from #13513 to 7.15.2 release notes (not a true backport)

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -57,6 +57,26 @@ This section summarizes the changes in the following releases:
...
 === Logstash 7.15.2 Release Notes
 
+[[known-issue-7-15-2]]
+==== Known issue
+
+Release 7.15.2 includes filter versions that will not run on JDK 8.
+This issue is fixed in the latest versions: 
+
+* https://www.elastic.co/guide/en/logstash-versioned-plugins/current/v3.1.12-plugins-filters-date.html[date filter 3.1.12]
+* https://www.elastic.co/guide/en/logstash-versioned-plugins/current/v7.2.7-plugins-filters-geoip.html[geoip filter 7.2.7]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:67]: Does the lockfile affect which versions can be manually updated? That is, can a user running 7.15.2 upgrade to these plugin versions, or do we need to suggest an upgrade to 7.16.1 as well.

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -57,6 +57,26 @@ This section summarizes the changes in the following releases:
...
+You can <<updating-plugins,update these plugins>> as usual:
+
+[source,sh]
+-----
+bin/logstash-plugin update logstash-filter-date
+bin/logstash-plugin update logstash-filter-geoip
+-----
+
+TIP: Clear the `JAVA_HOME` and `LS_JAVA_HOME` environment variables to run Logstash with the bundled JDK (currently JDK 11). 
+This approach is recommended and can help you avoid version incompatibilities.
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:78]:
```suggestion
TIP: Clear the `JAVA_HOME` environment variable to run Logstash with the bundled JDK (currently JDK 11). 
This approach is recommended and can help you avoid version incompatibilities. 
```

Comment (karenzone) [reply docs/static/releasenotes.asciidoc:78]: Looks like LS_JAVA_HOME wasn't added until 7.16 based on [this info](https://github.com/elastic/logstash/issues/12725), and we should remove the reference to it.

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (yaauie) [reply docs/static/releasenotes.asciidoc:67]: A user who downloads 7.15.2 could certainly install the updated versions of the plugins, in the usual way.

The lockfile is a snapshot of the plugins that are _currently_ installed, including all of their dependencies, and the plugin manager uses this info when installing or updating plugins to ensure that the dependency graph can be satisfied (and writes a new lockfile when it is done).

**Review by yaauie — APPROVED** [review]
LGTM, with the suggestion to remove references to `LS_JAVA_HOME`, as that was introduced in 7.16.0

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (karenzone) [reply docs/static/releasenotes.asciidoc:67]: Nice explanation, thank you!  I guess I've never thought about this before. Good to know. 😄

**Pull request merged**